src/screens/SearchScreen.tsx

```diff
@@ -49,6 +61,14 @@ export default function SearchScreen({
           onChangeText={onSearchChange}
           value={searchQuery}
         />
+        <Button
+          onPress={() => {
+            setActive(!active);
```

Comment:
you need to run refreshTrees in a useEffect, setActive is an async function and will not complete by the time you refresh